client/jetpack-connect/index.jsx

```diff
@@ -158,20 +155,21 @@ const JetpackConnectMain = React.createClass( {
 		}
 
 		if ( this.state.waitingForSites && ! this.props.isRequestingSites ) {
+			// eslint-disable-next-line react/no-did-update-set-state
```

Comment:
This usage seems safe for `setState` within `componentDidUpdate`, but proposals for a better solution are welcome.